src/sqlancer/questdb/QuestDBErrors.java

```diff
@@ -21,12 +22,34 @@ public static List<String> getExpressionErrors() {
         errors.add("too few arguments for 'in'");
         errors.add("cannot compare TIMESTAMP with type"); // WHERE column IN with nonTIMESTAMP arg
         errors.add("constant expected");
+        errors.add("Invalid date");
+        errors.add("invalid timestamp");
+        errors.add("invalid column");
+        errors.add("expression type mismatch");
+        errors.add("inconvertible value");
+        errors.add("not implemented: dynamic pattern would be very slow to execute");
+        errors.add("Cannot invoke \"java.lang.CharSequence.length()\" because \"seq\" is null");
```

Comment:
This seems like a potential bug, since Java error messages should presumably never surface to the user?